elements/pfe-autocomplete/src/pfe-autocomplete.scss

```diff
@@ -1,71 +1,68 @@
 @import "../../pfe-sass/pfe-sass";
+$LOCAL: autocomplete;
+
 :host {
-  display: block;
-  position: relative;
-}
+  --pfe-autocomplete--BoxShadow: pfe-var(box-shadow, inset);
+  --pfe-autocomplete--BackgroundColor: pfe-color(surface--lightest);
+  --pfe-autocomplete--Border:  #{pfe-var(border-width, ui)}  #{pfe-var(border-style, ui)}  #{pfe-color(surface--border)};
 
-.input-box-wrapper {
+  display: block;
   position: relative;
-  display:flex;
+  font-family: #{pfe-var(font-family)};
 }
 
 .sr-only {
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
-  border: 0;
-}
-
-#input-box-wrapper ::slotted(input) {
-  width: 100%;
-  flex: 1;
-  box-shadow: inset 0 0px 0px rgba(0,0,0,0.075) !important;
-  padding-left: 10px;
-  padding-right: 30px;
-  border-radius: 0;
-  background-color: #fff;
-  border: 1px solid #{pfe-color( surface--border)};
-  font-size: 16px; /*1rem*/
-  height: 40px;
-  transition: border-color ease-in-out 0.15s,box-shadow ease-in-out 0.15s;
-  opacity: 1;
-  outline: 0;
-}
-
-#input-box-wrapper ::slotted(input:disabled),
-button:disabled {
-  cursor: not-allowed;
-  background-color: transparent;
-  color: #ccc;
-  opacity: 0.5;
-}
-
-#input-box-wrapper button:focus,
-#input-box-wrapper ::slotted(input:focus) {
-  border-color: #66afe9;
-  outline: 0;
-}
-
-#input-box-wrapper ::slotted(input),
-button {
-  -webkit-appearance: none;
-}
-
-#input-box-wrapper ::slotted([type="search"]::-ms-clear) {
-  display: none;
-}
-
-#input-box-wrapper ::slotted(input[type="search"]::-webkit-search-cancel-button),
-#input-box-wrapper ::slotted(input[type="search"]::-webkit-search-decoration) {
-  -webkit-appearance: none;
+  @include pfe-sr-only;
+}
+
+#input-box-wrapper {
+  ::slotted(input) {
+    width: 100%;
+    flex: 1;
+    box-shadow: #{pfe-local(BoxShadow)} !important;
```

Comment:
Should this be the inset one too?

Reply:
this is ok, BoxShadow is the prop, & the name of the local var. The value of the local var includes "inset"